build.sh

```diff
@@ -29,12 +29,17 @@ build hugo-node h0.110.0-n18.13.0
 build hugo-node h0.76.5-n12.22.1 latest
 
 build drupal-node d9.5.10-n18.17.1 latest
+# build_arg <image_name> <tag> [build-args] [latest]
+build_arg drupal-node d9.5.10-n18.18.2 "--build-arg DRUPAL_VERSION=9.5.10 --build-arg NODE_VERSION=v18.18.2"
```

Comment:
I suggest we now make this the latest.

We use the latest tag in our Drupal sites. If we set this to the latest, we will automatically upgrade on the next rebuild.

//cc @ericpoirier @theodoreb

Reply:
@fredg02 This is great! Thanks for making these changes which I believe will make it easier for us to upgrade.

Reply:
Fixed :white_check_mark: